packages/venia-concept/src/components/ProductFullDetail/ProductFullDetail.js

```diff
@@ -50,7 +50,9 @@ class ProductFullDetail extends Component {
                     file: string.isRequired
                 })
             ),
-            description: string
+            description: shape({
+                html: string.isRequired
+            })
```

Comment:
```suggestion
            }).isRequired
```

Because you're accessing `description.html`, `render()` will throw if it's not an object.

Reply:
Added